src/middleware/navigation/checks/has.presenter.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { logger } from '../../../utils/logger';
+import { getApplicationData } from "../../../utils/application.data";
+import { checkPresenterDetailsEntered, NavigationErrorMessage } from '.././check.condition';
+
+export const checkIfPresenterDataExists = (req: Request, res: Response, next: NextFunction, redirectUrl: string): void => {
+  try {
+    if ( !checkPresenterDetailsEntered(getApplicationData(req.session)) ) {
```

Comment:
remove spaces inside condition like this `if (!checkPresenterDetailsEntered(getApplicationData(req.session)))`